bin/arnold

```diff
@@ -182,34 +182,34 @@ OPTIONS:
 
 COMMANDS:
 
-> Prepare current project:
+> Prepare current environment:
```

Comment:
I think there might be a confusion between what is called here an `environment` and the `env_type`. Does an `environment` means a customer/env_type combination to you?

If yes, I think we should still call it a namespace as it corresponds to a customer/env_type couple.

Reply:
Yes, it means a customer/env_type combination to me.
Ok, I'll do what you recommend.

Reply:
Done :heavy_check_mark: 
I also updated the commit message body.